source/braille.py

```diff
@@ -352,8 +352,16 @@
 	controlTypes.STATE_CHECKED: _("( )"),
 }
 
-DOT7 = 64
-DOT8 = 128
+#: Cursor shapes
+CURSOR_SHAPES = (
+	# Translators: The description of a cursor shape.

```

Comment:
nit: The translators comment should be specific and say "braille cursor shape", as the translator might not know this is related to braille (they may not see/shouldn't have to figure this out from the source file name).